openid-sharedsignals-framework-1_0.md

```diff
@@ -1718,6 +1718,16 @@ state
 > OPTIONAL An opaque value provided by the Event Receiver when the event is
   triggered. This is a nested attribute in the event payload.
 
+subject
+
+> REQUIRED. The value of the `subject` field in a Verification Event MUST always
+  be set to have a simple value of type `opaque`. The `id` of the value MUST be
+  the `stream_id` of the stream being verified. If the Transmitter does not
+  support multiple streams, this value MUST be set to "0"
```

Comment:
Doesn't the Transmitter still assign a stream ID even when it only supports one stream? If it doesn't then all of our other endpoints are broken.

Reply:
Changed this to "none" to avoid the idea that a single-stream Transmitter must also name their only stream.